Portugues br.ini

```
@@ -1,7 +1,7 @@
 [Header]
 Language = "Português (BR)"
-Author = "Autor: R3VOWOOD"
-Author2 = "Ajudante: Ritinha"
+Author = "R3VOWOOD, Ritinha, Miepiee,"
+Author2 = "LiveLM"
```

```suggestion
Author = "R3VOWOOD, Ritinha, LiveLM,"
Author2 = ""
```
Don't really wanna attribute myself as an author, since I basically just checked basic stuff and not really any translation work.

---

oh, okay!